packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -136,11 +137,39 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
       }
     }
 
+    if (credential.app?.categories.includes(AppCategories.crm)) {
+      const metadata = EventTypeMetaDataSchema.parse(eventType.metadata);
+      const appSlugToDelete = credential.app?.slug;
+
+      if (appSlugToDelete) {
+        const appMetadata = removeAppFromEventTypeMetadata(appSlugToDelete, metadata);
+
+        await prisma.$transaction(async () => {
```

Comment:
We don't need a transaction here it seems

Reply:
This is interesting. The idea is that we should be updating all the event types as one unit but this isn't the case. This logic is similar for other event type apps. Should we follow up with a refactor as I think this is happening in other parts of the handler.

Reply:
Yeah we can fix it in a followup